CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 # Contributing
 
-We'd love to have you join our group of contributors! Please e-mail your areas of interest and your availability to Alice (alice@code.org), and we’ll be happy to match you with a project. You can start setting up with these next steps.
+We'd love to have you join our group of contributors! Please e-mail your areas of interest and your availability to Alice (alice@code.org), and we’ll be happy to match you with a project.
```

Comment:
Is it possible to use an alias (mailing group?) rather than a hard-coded user email?